src/app_uart.c

```diff
@@ -270,6 +271,12 @@ void app_uart_parser (void * p_data, uint16_t data_len)
                 uart_payload.params.ack.ack_state.state = RE_CA_ACK_ERROR;
             }
 
+            if (RE_CA_UART_SET_ALL == uart_payload.cmd)
+            {
+                m_uart_ack = true;
+                err_code |= app_ble_scan_start();
```

Comment:
Why starting the scan here? To apply the new settings or something else?

Reply:
@ojousima Yes, to apply new settings. On  the main  code the similar function `app_ble_scan_start` is called earlier than the received UART package has been processed.